app/Http/Controllers/RemindersController.php

```diff
@@ -2,11 +2,14 @@
 
 namespace App\Http\Controllers;
 
+use App\Notifications\ReminderDeletedNotification;
 use App\Reminder;
-use Illuminate\Support\Arr;
+use Illuminate\Auth\Access\AuthorizationException;
 use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Notification;
 use Symfony\Component\HttpFoundation\Response;
```

Comment:
When contributing to a project, it's important to stick with the convetions. We're using a "length first, alphabetic second" type of import sorting.